package day_46_exceptions;

public class TestTryCatchThrows {

    public static void main(String[] args) {


        System.out.println("Hi there");
        WaitClass.wait1(5000);

        System.out.println("Hi there again");

        System.out.println("==================================");

        System.out.println("Bye");
        ///WaitClass.wait2(5000);

    }
}
